pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+
+import EmptyState from '@pkg/components/EmptyState.vue';
+import SnapshotCard from '@pkg/components/SnapshotCard.vue';
+import { Snapshot } from '@pkg/main/snapshots/types';
+
+interface Data {
+  snapshotsPollingInterval: ReturnType<typeof setInterval> | null;
+  isEmpty: boolean;
+}
+
+interface Methods {
+  pollingStart: () => void,
+}
+
+interface Computed {
+  snapshots: Snapshot[]
+}
+
+export default Vue.extend<Data, Methods, Computed, never>({
+  components: { EmptyState, SnapshotCard },
+
+  data(): Data {
+    return {
+      snapshotsPollingInterval: null,
+      isEmpty:                  false,
+    };
+  },
+
+  computed: { ...mapGetters('snapshots', { snapshots: 'list' }) },
+
+  watch: {
+    snapshots: {
+      handler(neu) {
```

Comment:
I know this is a bit of a nitpick, but can we use a better name than `neu`? I can't say that I ever really understood what `neu` means and we just as easily name this something along the lines of `val` or `newVal`.

Reply:
😅 I think 'neu' is new in german, this is used since 'new' would be a reserved word.